src/Microsoft.Extensions.Primitives/project.json

```diff
@@ -17,17 +17,11 @@
   },
   "frameworks": {
     "net451": { },
-    "netcore50": {
-      "dependencies": {
-        "System.Resources.ResourceManager": "4.0.0",
-        "System.Runtime": "4.0.20",
-        "System.Threading": "4.0.10"
-      }
-    },
     "dotnet5.4": {
       "dependencies": {
-        "System.Runtime": "4.0.21-*",
-        "System.Resources.ResourceManager": "4.0.1-*"
+        "System.Diagnostics.Debug": "4.0.10",
```

Comment:
> Please do. Always forcing the latest is wrong now that they've RTMed.
 
Not true. Not using the latest is *always* wrong now that we have netstandard. I'll ping you in person and explain it.